pandas/tests/indexing/test_loc.py

```diff
@@ -1579,3 +1579,13 @@ def test_loc_setitem_dt64tz_values(self):
         s2["a"] = expected
         result = s2["a"]
         assert result == expected
+
+
+@pytest.mark.parametrize("klass", [DataFrame, Series])
```

Comment:
@jbrockmendel 

IIRC this test should be in pandas/tests/frame/indexing/test_loc.py per our new org.

Reply:
Was not quite sure where to place them, because of tests for DataFrame and Series. Would be happy to move them, if this place is better.

Reply:
This is the correct file, should probably be inside TestLabelSlicing class.

use `frame_or_series` fixture instead of `klass` here

Reply:
Done

Reply:
There is no class ``TestLabelSlicing`` or similar to this?